Src/Plugins/Security/SEC101_043.NuGetCredentialsValidator.cs

```diff
@@ -94,17 +94,20 @@ internal static List<string> ExtractHosts(string hostXmlAsString)
                     continue;
                 }
 
-                validationResults.Add(new ValidationResult
+                foreach ((string user, string password) in credentials)
```

Comment:
>password [](http://example.com/codeflow?start=46&length=8)

maybe this is the place to do the likely powershell variable check?

but why would there be a powershell variable in XML content?? :)

Reply:
removed the powershell check.